src/Analysim.Web/ClientApp/src/app/projects/project/project.component.ts

```diff
@@ -103,13 +103,13 @@ export class ProjectComponent implements OnInit {
               this.latestVersion = this.versions[0]; // Default to the latest version
             }
             // console.log("the loatest version  : ", this.latestVersion);
+            this.projectService.getNotebookFile(result.notebooks.filter((notebook) => notebook.name.toLowerCase() === "readme")[0], this.latestVersion).subscribe(
+              notebookJson => {
+                this.notebookContent = this.sanitizer.bypassSecurityTrustHtml(this.renderNotebook(notebookJson));
+                // console.log("the notebook content is : ", this.notebookContent);
+              });
           });
           // console.log("readme file is : ", result.notebooks.filter((notebook) => notebook.name.toLowerCase() === "readme")[0]);
-          this.projectService.getNotebookFile(result.notebooks.filter((notebook) => notebook.name.toLowerCase() === "readme")[0], this.latestVersion).subscribe(
```

Comment:
this probably needs to stay because now it doesn't show notebooks with only one version.

Reply:
great, it works